skyportal/models/localization.py

```diff
@@ -235,6 +235,16 @@ class LocalizationTile(Base):
     healpix = sa.Column(healpix_alchemy.Tile, primary_key=True, index=True)
 
 
+LocalizationTile.__table_args__ = (
+    sa.Index(
+        'localizationtile_id_healpix_index',
+        LocalizationTile.id,
+        LocalizationTile.healpix,
+        unique=True,
+    ),
+)
```

Comment:
I think you can have the `__table_args__` added directly into the class definition, without having to call `LocalizationTile` and append it like that.

Reply:
@guynir42 Our other indices are done this way... maybe because they arent defined yet?

Reply:
Usually you define things like this if you need other tables to already exist to refer to them. It looks to me like this can work just as well inside the `class` block.

Reply:
Yeah LocalizationTile.id is not defined.